R/add_quantiles.R

```diff
@@ -0,0 +1,154 @@
+#' @title Add lines indicating the survival quantiles to a survival plot
+#'
+#' @description Method to add lines indicating the specified quantiles of the survival curve.
+#'
+#' @param gg visR object
+#' @param ... other arguments passed on to the method to modify \code{\link[ggplot2]{geom_line}}
+#'
+#' @examples
+#' 
+#' library(visR)
+#' 
+#' adtte %>%
+#'   estimate_KM("SEX") %>%
+#'   visr() %>%
+#'   add_quantiles() 
+#' 
+#' adtte %>%
+#'   estimate_KM("SEX") %>%
+#'   visr() %>%
+#'   add_quantiles(c(0.25, 0.50, 0.75)) 
+#' 
+#' adtte %>%
+#'   estimate_KM("SEX") %>%
+#'   visr() %>%
+#'   add_quantiles(c(0.25, 0.50, 0.75)) 
+#'
+#' @return Lines indicating the quantiles overlayed on a visR ggplot
+#'
+#' @rdname add_quantiles
+#'
+#' @export
+
+add_quantiles <- function(gg, ...){
+  UseMethod("add_quantiles", gg)
+}
+
+#' @param gg A ggplot created with visR
+#' @param quantiles y-axis values for which to draw the lines (default: 0.5)
+#' @param linetype aesthetic of ggplot2 \code{\link[ggplot2]{geom_line}} (default: dashed)
+#' @param linecolour aesthetic of ggplot2 \code{\link[ggplot2]{geom_line}} (default: black)
+#' @param alpha aesthetic of ggplot2 \code{\link[ggplot2]{geom_line}} (default: 0.8)
+#' @param ... other arguments passed on to the method to modify \code{\link[ggplot2]{geom_line}}
+#'
+#' @rdname add_quantiles
+#' @method add_quantiles ggsurvfit
+#' @export
+
+add_quantiles.ggsurvfit <- function(gg, 
+                                    quantiles = 0.5,
+                                    linetype = "mixed",
+                                    linecolour = "strata",
```

Comment:
if i want black, do i put "black"? this is not clear. Perhaps you can specify that this needs to be a character representing ....